src/services/posts.js

```diff
@@ -44,9 +44,20 @@ const getPost = async({id}) => {
 	return post.rows[0]
 }
 
+// UPDATE ARTICLES ENDPOINT
```

Comment:
this service is meant to be generic so should not have anything specific to an article. it's not really an endpoint too, it's just a function. 

if we want to document, let's look into the [jsdoc type](https://jsdoc.app/about-getting-started.html) which we can use to define what each parameter means

Reply:
yes please